Buildings/Fluid/FMI/FlowSplitter_u.mo

```diff
@@ -148,6 +148,13 @@ the model stops with an error.
 </html>", revisions="<html>
 <ul>
 <li>
+June 9, 2016, by Thierry S. Nouidui:
+</li>
```

Comment:
@tsnouidui This should be `<br/>` and not a new list item. See the other entries for formatting.